packages/lake/src/components/TabView.tsx

```diff
@@ -151,20 +169,68 @@ const styles = StyleSheet.create({
   },
 });
 
+type TabViewLinkProps = {
+  onChange: ((id: string) => void) | undefined;
+  tab: Tab;
+  children: ReactNode;
+  style: (state: PressableStateCallbackType & { active: boolean }) => TextProps["style"];
+  role?: WebRole;
+  activeTabId: string | undefined;
+  onFocus?: PressableTextProps["onFocus"];
+  onBlur?: PressableTextProps["onBlur"];
+  onPress?: () => void;
+  ariaCurrentValue?: "page" | "location";
+};
+
+const TabViewLink = forwardRef<Text, TabViewLinkProps>(
+  ({ children, style, tab, onChange, activeTabId, onBlur, onFocus, onPress }, ref) => {
+    return match(tab)
+      .with({ url: P.string }, ({ url }) => (
+        <Link ref={ref} to={url} style={style} onFocus={onFocus} onBlur={onBlur} onPress={onPress}>
+          {children}
+        </Link>
+      ))
+      .with({ id: P.string }, ({ id }) => {
+        const tabId = getTabId(tab);
+        const isActive = tabId === activeTabId;
```

Comment:
nit: `getTabId(tab)` call is useless since you are already in a branch where `tab` as an `id` and not an `url`. You can do `const isActive = id === activeTabId;`